src/components/TodoItem.js

```diff
@@ -2,27 +2,41 @@ import {Observable} from 'rx';
 import {button, div, input, label, li} from '@cycle/dom';
 import {propHook, ENTER_KEY, ESC_KEY} from '../utils';
 
+// THE TODO ITEM INTENT
+// This intent function returns a stream of all the different,
+// actions that can be taken on a todo.
 function intent(DOM) {
+  // THE INTENT MERGE
+  // Merge all actions into one stream.
   return Observable.merge(
+    // THE DESTROY ACTION STREAM
     DOM.select('.destroy').events('click').map(() => ({type: 'destroy'})),
-
+    // THE TOGGLE ACTION STREAM
     DOM.select('.toggle').events('change').map(() => ({type: 'toggle'})),
-
+    // THE START EDIT ACTION STREAM
     DOM.select('label').events('dblclick').map(() => ({type: 'startEdit'})),
-
+    // THE ESC KEY ACTION STREAM
     DOM.select('.edit').events('keyup')
       .filter(ev => ev.keyCode === ESC_KEY)
       .map(() => ({type: 'cancelEdit'})),
-
+    // THE ENTER KEY ACTION STREAM
     DOM.select('.edit').events('keyup')
       .filter(ev => ev.keyCode === ENTER_KEY)
       .merge(DOM.select('.edit').events('blur', true))
       .map(ev => ({title: ev.target.value, type: 'doneEdit'}))
-  ).share();
+  )
+  // MAKE THIS OBSERVABLE HOT
+  // By sharing it will act as a hot Observable.
+  .share();
 }
 
 function model(props$, action$) {
+  // THE SANITIZED PROPERTIES
+  // If the list item has no data set it as empty and not completed.
   const sanitizedProps$ = props$.startWith({title: '', completed: false});
+  // THE EDITING STREAM
+  // Create an editing stream that consists only of
+  // actions related to editing.
```

Comment:
Not exactly what it means. Because of the startWith(false), this stream doesn't have actions (actions are objects with the type and values... like in Redux). This stream emits booleans. It is a stream representing the "is editing" state.